### ISSUE TITLE
XCE and mx bits

### ISSUE THREAD

**Issue by jeffythedragonslayer** [undisbeliever/snesdev-notes#12]
I suggest rewording the following two lines about the XCE instruction:

> m set if enabling native mode
> x set if enabling native mode

because the m and x bits are not going from clear to set when enabling native mode, they already were forced to be set when the processor booted up in emulation mode.  Source:

http://6502.org/tutorials/65c816opcodes.html#6.10.4

**Comment by undisbeliever** [comment]
That was a typo.  I had meant to write "set if enabling emulation mode".

I have rewritten the XCE psuedo-code to use an if statement, since I also forgot XCE can change the stack pointer.

Thank you for reporting this.
